apps/docs/src/app/platform/component-docs/platform-link/platform-link-header/platform-link-header.component.html

```diff
@@ -1,8 +1,12 @@
-<header>Link</header>
+<header>Link Component</header>
 <description>
-    The Link Component is used to enhance standard HTML link.
-    <br /><br />
-    Link can be clicked by keyboard enter key and Mouse click.
+    Link in Platform is functionally similar to the Link in
+    <a href="https://sap.github.io/fundamental-ngx/#/core/link">fundamental-ngx/core</a>. In Platform, Link is wrapped
```

Comment:
this is fine, just fyi, you can even use `<a [routerLink]="'/core/link'">` instead

Reply:
Done.